.travis.yml

```diff
@@ -34,6 +34,22 @@ matrix:
       env: TOXENV=py36-accept
     - python: 3.6
       env: TOXENV=py36-examples
+    - python: 3.7
+      env: TOXENV=py37-local
+      dist: xenial
```

Comment:
Is this required because 3.7 specifically? Can we doc or otherwise track the special case somehow?

Reply:
We should document it somewhere, yes. It's currently a necessary workaround for Travis' build configurations to get Python 3.7.